teensytoany/teensytoany.py

```diff
@@ -301,7 +308,17 @@ def i2c_write_payload(self, address: int, payload: Sequence) -> None:
                 f"0x{address:02x} 0x{register_address:04x} 0x{data:04x}")
         else:
             raise NotImplementedError()
+        """
+
+    def i2c_read_payload(self, address: int, register_address: int, num_bytes: int) -> Sequence:
+
+        cmd = f"i2c_read_payload 0x{address:02x} 0x{register_address:02x} {num_bytes}"
+        returned = self._ask(cmd)
+        register_data = [int(val, base=0) for val in returned.split()] # returns big endian
+        return register_data
 
+    # Obsolete
```

Comment:
Similarly to the write path, this code should be kept in as a fall back.

Generally, when code is to be removed, it should be removed outright. Leaving orphaned code is a bad idea.